fedimint-core/src/module/mod.rs

```diff
@@ -364,13 +367,6 @@ impl ApiEndpoint<()> {
         E::Param: Debug,
         E::Response: Debug,
     {
-        #[instrument(
-            target = "fedimint_server::request",
-            level = "trace",
-            skip_all,
-            fields(method = E::PATH),
-            ret,
-        )]
```

Comment:
you can do `fields(id = REQ_ID.fetch_add(1, Ordering::SeqCst))` btw

Reply:
or is it instrument not supporting CONST target

Reply:
It was actually the `ID`, thanks, but while at it, I can also use const target, so let's just keep it this way.

Reply:
you can even do like 

```
fields(id = {
  static REQ_ID: AtomicU64 = AtomicU64::new(0); 
  REQ_ID.fetch_add(1, Ordering::SeqCst)) 
})